app/Http/Livewire/Components/Market/MarketComponent.php

```diff
@@ -0,0 +1,204 @@
+<?php
+
+namespace App\Http\Livewire\Components\Market;
+
+use App\Models\Trade;
+use Illuminate\Support\Carbon;
+use Illuminate\Support\Facades\DB;
+use Livewire\Component;
+
+class MarketComponent extends Component
+{
+    public $isCreateModalOpen = false;
+    public $isRespondModalOpen = false;
+
+    public $trades;
+    public $trade;
+
+    public $paginator = [];
+    public $page = 1;
+    public $itemsPerPage = 15;
+    public $totalItems = 0;
+
+    public $bounds = [
+        'units_per_hour'    => 0,
+        'duration'          => 0,
+        'total_volume'      => 0,
+        'price_per_unit'    => 0,
+        'mix_co2'           => 0,
+    ];
+
+    public $filter = [
+        'hydrogen_type'     => [],
+        'units_per_hour'    => '',
+        'duration'          => '',
+        'total_volume'      => '',
+        'price_per_unit'    => '',
+        'mix_co2'           => '',
+        'trade_type'        => [],
+    ];
+
+    protected $listeners = ['listingCreated' => 'listingCreated'];
+
+    public function mount()
+    {
+        $this->updateTrades(true);
+    }
+
+    public function render()
+    {
+        return view('livewire.components.market.market-component')->extends('layouts.app');
+    }
+
+    public function openCreateModal()
+    {
+        $this->isCreateModalOpen = true;
+    }
+
+    public function closeCreateModal()
+    {
+        $this->isCreateModalOpen = false;
+    }
```

Comment:
je kunt dit soort dingen korter maken door er zoiets mee te doen:

`   function toggleIsCreateModal(){
            $this->isCreateModalOpen = !$this->isCreateModalOpen;
        }`

Reply:
Helemaal waar, vrij logisch ook. Misschien moet ik na het developen nog een extra pass doen om dit soort dingen even op te ruimen.